src/seedu/addressbook/AddressBook.java

```diff
@@ -450,7 +450,7 @@ private static String getMessageForSuccessfulAddPerson(String[] addedPerson) {
      * @return feedback display message for the operation result
```

Comment:
Mentioned during the tutorial, you should update the header comments to include what you have changed. Furthermore, since the find is now case insensitive, regression testing would fail, and should be changed accordingly.